File: pgm/ABC_Java/ABC_345_A.java
import java.io.*;
// import java.math.*;
import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String S = reader.readLine();
        final String Yes = "Yes";
        final String No = "No";
        final String lt = "<";
        final String eq = "=";
        final String gt = ">";
        System.out.println (
            lt.equals (S.substring (0, 1)) &&
                    String.join ("", Collections.nCopies (S.length() - 2, eq))
                        .equals (S.substring (1, S.length() - 1)) &&
                    gt.equals (S.substring (S.length() - 1, S.length()))
                ? Yes
                : No);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_345_A.kt
// import java.time.*
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder

// import java.util.StringTokenizer

// import kotlin.math.*

const val lt: String = "<"
const val eq: String = "="
const val gt: String = ">"
const val Yes: String = "Yes"
const val No: String = "No"
const val one: Int = 1

fun main() {
    val S: String = readln()
    if (S.startsWith(lt) &&
                    S.endsWith(gt) &&
                    S.substring(one, S.length - one).all { it == eq.single() }
    ) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}
